pkg/client/rootd/dns/server.go

```diff
@@ -155,6 +155,9 @@ func (s *Server) shouldDoClusterLookup(query string) bool {
 		case n == 2 && strings.HasPrefix(query, wpadDot):
 			// Reject "wpad.<namespace>."
 			return false
+		case n == 3 && strings.HasSuffix(query, ".svc"):
+			// Accept "<service-name>.<namespace>.svc"
+			return true
```

Comment:
I'm afraid this won't work. For two reasons really.
1. At this point, the `query` will always end with a dot.
2. On most platforms, the query will never end up here because the suffix isn't known to be a top-level domain handled by the Telepresence's DNS resolver. Consequently, the DNS system will never delegate the query to us.

A fix that will work is to add "svc" as a mandatory entry to the `includeSuffixes` list. That way the macOS specific [updateResolverFiles](/telepresenceio/telepresence/blob/release/v2/pkg/client/rootd/dns/server_darwin.go#L187) and systemd-resolved specific [updateLinkDomains](/telepresenceio/telepresence/blob/release/v2/pkg/client/rootd/dns/resolved_linux.go#L102) will tell the OS resolver that it's handled by our resolver.